exalearn-ui/src/app/pages/main-page/main-page.component.scss

```diff
@@ -52,4 +52,12 @@
 	min-width: 300px;
 	height: 8vh;
 	margin: 10px 0 10px;
+	a {
+		color: $white;
```

Comment:
Same as above, set up specific class to such `<a>` tag (for instance 'routing-link', but it's up to you)